frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -46,60 +46,86 @@ export class ImportWizard extends Wizard {
   /**
    * Edits attributes of a NICs that are being imported from source VM.
    */
-  async updateImportedNICs() {
+
+  async editNic(nic: Network) {
+    const nicDialog = new NetworkInterfaceDialog();
+    await clickKebabAction(nic.name, KEBAP_ACTION.Edit);
+    await waitForNoLoaders();
+    const networks = await nicDialog.getNetworks();
+    if (networks.length > 0) {
+      await nicDialog.selectNetwork(networks[networks.length - 1]);
+      await nicDialog.fillName(nic.name);
```

Comment:
why are we filling out the name? it should already have this value

Reply:
Removed it. That was leftover, cause in previous versions NIC name was incorrect and had to be replaced, now there is no need in it.

Reply:
still there

Reply:
@ibragins maybe you forgot to push the newest changes?

Reply:
Now it is fine